### PR TITLE
added installation updates and review hooks

### PR DESCRIPTION
**Pull request by ruhmamehek**

### CODE REVIEW

File: docs/docs/getting-started/integrating_customisations.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+- Register the customised component's `id` and corresponding import to `tcx extensions`
+
+# Building the customised code
+
+- Run `npm run build` in the root directory. You will get 3 files in the directory, `dist/`:
+    - `build.css`
+    - `guides-extension.js`
+    - `guides-extension.umd.cjs`
+
+- Go to `CRXDE`
```
Comment (ishank-s) [inline docs/docs/getting-started/integrating_customisations.md:29]: add crxde sample path here

Comment (ishank-s) [reply docs/docs/getting-started/integrating_customisations.md:29]: cloud users dont have access to crxde, we need to mention solution for cloud here as well

Comment (ruhmamehek) [reply docs/docs/getting-started/integrating_customisations.md:29]: Could you help me with the cloud solution please?

Comment (ishank-s) [reply docs/docs/getting-started/integrating_customisations.md:29]: better yet refernce link on how to do this for Customer's AEM instance (cloud or on prem)

**Review by ishank-s — APPROVED** [review]

**Pull request merged**